packages/riipen-ui/src/components/Popover.jsx

```diff
@@ -280,6 +347,11 @@ class Popover extends React.Component {
       this.handleClose();
     } else if (event.type === "keydown" && event.key === "Escape") {
       this.handleClose();
+    } else if (event.type === "keydown" && this.props.handleKeyDown) {
+      // stop this event from triggering other keydown listeners
+      event.stopImmediatePropagation();
```

Comment:
need to stop this event from triggering other keydown listeners, otherwise when 'changing tab' in `InteractiveTooltip`, all `Popover` `keydown` listeners fire, causing the tab to change `numberOfPopovers` times